theme/styles/mixins/flex.scss

```diff
@@ -0,0 +1,46 @@
+@charset "UTF-8";
```

Comment:
důvod specifikace charsetu?

Reply:
Kvůli poznámkám nahoře. Nic se nestane, když tam nebude, ale je to správná practice ho tam mít, protože se může stát, že se potom něco špatně vygeneruje.